packages/apps/human-app/server/src/modules/job-assignment/job-assignment.controller.ts

```diff
@@ -13,10 +12,10 @@ import { InjectMapper } from '@automapper/nestjs';
 import { Mapper } from '@automapper/core';
 import { JobAssignmentService } from './job-assignment.service';
 import {
-  JobAssignmentCommand,
   JobAssignmentDto,
-  JobsAssignmentParamsDto,
+  JobAssignmentCommand,
   JobAssignmentResponse,
+  JobsAssignmentParamsDto,
```

Comment:
imo this naming convention has two flaws: 
1. Is to similar by usecase
2. JobsAssignment... don't tell about real usecase - maybe: JobsFetch Dto/Params/Command/Response,

Reply:
Fixed in: a1b51198